rpcs3/rpcs3_app.cpp

```diff
@@ -168,30 +168,38 @@ void rpcs3_app::InitializeCallbacks()
 
 		bool disableMouse = guiSettings->GetValue(gui::gs_disableMouse).toBool();
 
+		// Make display geometry rectangle for the gs frame
+		// Take area of main window & center it neatly inside of it
+		// This prevents the frame being spawned wherever your cursor is, which is really annoying for multi-screen users
+		QRect main_window_geometry = RPCS3MainWin->geometry();
+		QRect gs_frame_geometry = QRect(main_window_geometry.left() + ((main_window_geometry.width() - w) / 2),
+										main_window_geometry.top() + ((main_window_geometry.height() - h) / 2),
```

Comment:
btw, using many tabs misaligns on any other setup
maybe do it like this:
![image](https://user-images.githubusercontent.com/23019877/34603122-e1cd4c6c-f202-11e7-9bd3-ac804d0a367b.png)